test/functional/legacy/038_virtual_replace_spec.lua

```diff
@@ -10,6 +10,8 @@ describe('Virtual replace mode', function()
   it('is working', function()
     -- Make sure that backspace works, no matter what termcap is used.
     execute('set t_kD=x7f t_kb=x08')
+    -- Use vi default for 'smarttab'
+    execute('set nosmarttab')
```

Comment:
Well, technically, vim's default here is to use vi's default, but I can change it if its clearer the other way.